docs/rfcs/UserProfilesv2.md

```diff
@@ -0,0 +1,145 @@
+# User Profiles (v2)
+
+This document describes the Mozilla IAM user profiles version 2.
+
+IAM Goals:
+- MUST stay compatible with v1 (current profiles)
+- Add structured access information that can be reliably queried
+- Integrate the [Automatic Access Expiration](AutomaticAccessExpiration.md) to the access information
+- Allow for identity information to be transported from social authentication providers to our profile
+- Standardize documentation around the profile
+- Allow for a standardized API on top of the user profile data ([Person-API](https://github.com/mozilla-iam/person-api))
+- Version the profile for possible future updates
+
+## Use cases
+
+1. Machines such as TaskCluster want to reliably query Person-API and get authoritative answers about the user's group
+   membership. The groups must be easy to query with no ambiguity (i.e. no string-namespacing).
+2. User managing GitHub wants to access GitHub user_ids (not email) from logged in users in order to query the GitHub
+   API and get relevant data about the user.
+3. The access provider wants to expire access when it is no longer used (Automatic Access Expiration).
+4. Security team wants to assert that access is lost to certain services (RPs) when they're no longer used by the person
+   who used to have access.
+
+## Profile proposal
+
+**Main changes**
+
+1. `_schema` field contains the URL to the json-schema.org validation file. The validation file itself contains the
+   revision-equivalent, as well as the environment the schema is written for directly in the URL. This field also
+   mirrors it, eg: "person-api.sso.allizom.org" => dev. environment, "v2" is version 2.
+2. `emails` structure becomes `identities`. This is a dangerous change as it removes a previously existing field. After
```

Comment:
You could also just duplicate the property, and document the "emails" property as deprecated...

But it does look like over kill in this case...

Reply:
yeah id rather have as little deprecated fields as possible for simplicity, if possible